saleor/product/models.py

```diff
@@ -81,7 +81,7 @@ class ProductType(models.Model):
         'Attribute', related_name='product_variant_types', blank=True)
     is_shipping_required = models.BooleanField(default=False)
     tax_rate = models.CharField(
-        max_length=128, default=DEFAULT_TAX_RATE_NAME, blank=True,
+        max_length=128, blank=True,
```

Comment:
I think Django generates migrations for such changes, if yes, we should provide them with in PR.